plugins/promdump.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: promdump
+spec:
+  version: v0.2.2
+  homepage: https://github.com/ihcsim/promdump
+  shortDescription: promdump dumps the head and persistent blocks of Prometheus.
+  description: |
+    promdump dumps the head and persistent blocks of Prometheus. It supports
+    filtering the persistent blocks by time range.
+
+    promdump is a tool that can be used to dump Prometheus data blocks. It is
+    different from the 'promtool tsdb dump' command in such a way that its output
+    can be re-used in another Prometheus instance. And unlike the Promethues TSDB
+    'snapshot' API, promdump doesn't require Prometheus to be started with the
+    '--web.enable-admin-api' option. Instead of dumping the entire TSDB, promdump
+    offers the flexibility to filter persistent blocks by time range.
+  platforms:
+  - selector:
+      matchLabels:
+        os: darwin
+        arch: amd64
```

Comment:
`darwin`+`arm64` has become quite popular. Please consider distributing binaries for that platform too.